### DISCUSSION TITLE
Rename Camera for Windows ?

### DISCUSSION THREAD

**Discussion by schnibbi** [geerlingguy/pi-webcam#31] [Ideas]
Is there any possibility to give the camera another name than UVC Camera ?

I looked up all files in rep but couldnt finde any hint for the camera name in windows

**Comment by geerlingguy** [comment]
It's called Pi Webcam for me... the string comes from the uvc-gadget repo that is cloned via this project, as configured in config.yml.

**Reply by schnibbi** [reply]
If i got it right you mean i have to alter anything in your toher git rep: https://github.com/geerlingguy/uvc-gadget.git ?
So what line is it then, i cant find anything that says UVC Camera, as it is called in my applications on windows. in windows itself its calls PI Webcam. trying to identify for example on lightburn, it only tells UVC Camera
